web/src/user_profile.js

```diff
@@ -298,8 +298,7 @@ function initialize_user_type_fields(user) {
 }
 
 export function show_user_profile(user, default_tab_key = "profile-tab") {
-    popovers.hide_all();
-
+    hideAll();
```

Comment:
I don't love importing this without namespace; it's not very readable, since you have no idea what we're hiding.

I think still exporting a `popover_menus.hide_all()` that just wraps `tippy.hideAll()` would probably be a good plan.

And then we can just mechanically convert in most call points `popovers.hide_all()` into `popover_menus.hide_all(); sidebar_ui.hide_narrow_width_sidebars();` or some API along those lines, and `popovers.hide_all_except_sidebars` into `popover_menus.hide_all()` as a bulk change that is very low risk -- and then follow that up with individually verifying places we can remove some of that logic.

(The best approach would probably be a commit introducing those new interfaces and converting all the call points where nothing interesting is happening to do those things, and then have much smaller commits for places where we actually want to delete the hiding logic because it's unnecessary, with each commit having a specific reason in the commit message)